helix-core/src/main/java/org/apache/helix/util/RebalanceUtil.java

```diff
@@ -177,12 +177,8 @@ public static void scheduleOnDemandPipeline(String clusterName, long delay) {
 
   public static void scheduleOnDemandPipeline(String clusterName, long delay,
       boolean shouldRefreshCache) {
-    if (clusterName == null) {
-      LOG.error("Failed to issue a pipeline run. ClusterName is null.");
-      return;
-    }
-    if (delay < 0L) {
-      LOG.error("Failed to issue a pipeline run. Delay is invalid.");
+    if (clusterName == null || delay < 0L) {
+      LOG.warn("ClusterName is null or delay is invalid, skip the pipeline issuing.");
```

Comment:
It's OK to merge them. But print them out. Otherwise, when debug, it will be hard for us to understand which one cause it.